test/fw/ptl/utils/pbs_dshutils.py

```diff
@@ -2071,55 +2073,65 @@ def create_temp_dir(self, hostname=None, suffix='', prefix='PtlPbs',
         :type suffix: str
         :param prefix: the directory name will begin with this prefix
         :type prefix: str
-        :param dir: the directory will be created in this directory
-        :type dir: str or None
-        :param uid: Optional username or uid of temp directory owner
-        :param gid: Optional group name or gid of temp directory
+        :param dirname: the directory will be created in this directory
+        :type dirname: str or None
+        :param asuser: Optional username of temp directory owner
+        :type asuser: str
+        :param asgroup: Optional group name of temp directory
                     group owner
+        :type asgroup: str
         :param mode: Optional mode bits to assign to the temporary
                      directory
+        :type mode: octal integer
         :param level: logging level, defaults to INFOCLI2
         """
+        current_user_info = self.get_id_info(self.get_current_user())
+        uid = current_user_info['uid']
+        gid = current_user_info['gid']
+        if asuser is not None:
+            uid = self.get_id_info(str(asuser))['uid']
+        if asgroup is not None:
+            gid = grp.getgrnam(str(asgroup))[2]
```

Comment:
Same as above, currently PbsGroup doesn't have get_group method, but you can introduce it, it will be useful in future also to someone